pixi.toml

```diff
@@ -6,6 +6,9 @@ platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
 [tasks]
 postinstall = "pip install --no-build-isolation --no-deps --disable-pip-version-check -e ."
 coverage = "pytest tests --cov=metalearners --cov-report=xml --cov-report term-missing --color=yes"
+# Note that this task will fail on systems without bash.
+nb-convert = "nb-convert.sh"
```

Comment:
this probably won't work on windows unless cygwin/msys2 is installed but i guess this is fine

Reply:
Yup, that's what I was trying to allude to with the commend above... :/

Reply:
oh i overlooked this comment 😅